Bottom: 2d736f5e5bc6386de61ffa8032d86667dab64a85
Top:    4bfddc5ac9db7105c8b70af73856320bd1e960ae
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 17:05:13 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 8f6e893..ca691e0 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -331,13 +331,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
